mllib/src/main/scala/org/apache/spark/ml/feature/PCA.scala

```diff
@@ -106,7 +107,7 @@ object PCA extends DefaultParamsReadable[PCA] {
 
 /**
  * :: Experimental ::
- * Model fitted by [[PCA]].
+ * Model fitted by [[PCA]].  Transforms vectors to a lower dimensional space.
```

Comment:
(Trivial.. but maybe single space between `* Model fitted by [[PCA]]` and `Transforms vectors to a lower dimensional space.`)